src/Npgsql/NpgsqlWriteBuffer.cs

```diff
@@ -122,16 +117,9 @@ public async Task Flush(bool async, CancellationToken cancellationToken = defaul
             CancellationTokenSource? combinedCts = null;
 
             var finalCt = cancellationToken;
-            if (async && Timeout > TimeSpan.Zero)
+            if (async && Timeout > System.Threading.Timeout.InfiniteTimeSpan)
```

Comment:
`Timeout` condition can be removed too in that case.

Reply:
Yeah, I agree that no current use of the `NpgsqlWriteBuffer` class sets the `Timeout` after object initialization but that's not the case for `NpgsqlReadBuffer` and I'd rather keep the symmetry between them.